go.mod

```diff
@@ -282,3 +282,5 @@ replace (
 	go.opentelemetry.io/otel/log => go.opentelemetry.io/otel/log v0.3.0
 	go.opentelemetry.io/otel/sdk/log => go.opentelemetry.io/otel/sdk/log v0.3.0
 )
+
+replace github.com/moby/buildkit => github.com/jedevc/buildkit v0.8.2-0.20241010144035-f69c0798296a
```

Comment:
This points to a custom fork right now (so as to avoid pulling in all the protobuf changes before they've actually been battle-tested in a release - also I suspect it'll be a bit of a complex change :cry:)

We should point it to our fork at https://github.com/dagger/buildkit before merging.